### ISSUE TITLE
Setup.bash

### ISSUE THREAD

**Issue by iLeandro** [DefectDojo/django-DefectDojo#2251]
**Bug description**
When running setup.bash (inside setup folder) I get two errors. Requirements not found (requirements is in up folder) and django module not found

**Steps to reproduce**
Clone Repository
Enter setup folder
run setup.bash (./setup.bash)
Choose option 1/2 (Single Server / Dev Install - both will give the same error)
Get error regarding requirements.txt not found and manage.py no module named django

**Expected behavior**
Should end the script with no error and let me run python3 manage.py runserver

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [ ] Docker
- [X] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Ubuntu 18.04
 - DefectDojo Commit Message: 1a24a430: Merge pull request #2228 from 10Security/master [2020-04-23 14:28:49 -0400]

I don't have virtualenv setup, don't know if I need.
When I run the docker way (docker-compose build and up) it works perfectly but I don't want Docker instances. I want all in one place right now.

**Comment by dayal18** [comment]
It's not a bug.. you'll need to follow the setup procedure..

do these steps:

cd /opt
mkdir dojo
cd dojo
git clone https://github.com/DefectDojo/django-DefectDojo.git
cd django-DefectDojo

django-DefectDojo# setup/setup.bash
...
...
..
After installation run this

django-DefectDojo# pip3 install -r requirements.txt
..
..
django-DefectDojo# python3 manage.py runserver 0.0.0.0:8000

**Comment by Chan9390** [comment]
Doing a `pip3 install -r requirements.txt` and `python3 manage.py runserver 0.0.0.0:8000` runs the server but when I visit the website I get this,

![image](https://user-images.githubusercontent.com/12944530/82135614-3ee92c00-9822-11ea-898b-b36fb1d12d82.png)

Did you face this error @dayal18  ?

**Comment by iLeandro** [comment]
> Doing a `pip3 install -r requirements.txt` and `python3 manage.py runserver 0.0.0.0:8000` runs the server but when I visit the website I get this,
> 
> ![image](https://user-images.githubusercontent.com/12944530/82135614-3ee92c00-9822-11ea-898b-b36fb1d12d82.png)
> 
> Did you face this error @dayal18 ?

Try to run `python3 manage.py collectstatic`

Cheers

**Comment by Chan9390** [comment]
Thank you @iLeandro for the reply

I found that you dont even need to do a `pip3 install -r requirements.txt` again, because its already done during the setup.

There is a custom python3 binary at the location `/opt/dojo/bin/python3`, so while starting defectdojo I would do this `/opt/dojo/bin/python3 manage.py runserver 0.0.0.0:8000`

Running code from `release/1.6.0` branch solved the static files issues as well (as Node 10.x is used instead of deprecated 6.x)

**Comment by dayal18** [comment]
@Chan9390 yes, you'll have to change the Node version from 6 to 10.. else the static command won't execute. Glad it worked.
